### PR TITLE
Prevent total_latency_ variable overflow

### PR DESCRIPTION
**Pull request by sageig**
Change total latency calculation in subscriber_impl.h from simple addition to exponential moving average in order to prevent overflow.

### CODE REVIEW

**Comment by matt-attack** [comment]
Looks good to me!

File: swri_roscpp/include/swri_roscpp/subscriber_impl.h [hunk 1]
```diff
@@ -81,7 +81,8 @@ class SubscriberImpl
         } else {
           min_latency_ = std::min(min_latency_, latency);
           max_latency_ = std::max(max_latency_, latency);
-          total_latency_ += latency;
+          double latency_weight_ = 0.1;
+          total_latency_.fromSec(latency_weight_*latency.toSec() + (1-latency_weight_)*total_latency_.toSec());
```
Comment (matt-attack) [inline swri_roscpp/include/swri_roscpp/subscriber_impl.h:85]: Actually, it would probably be good to change that 1 to a 1.0 to make sure it is a double

Comment (pjreed) [reply swri_roscpp/include/swri_roscpp/subscriber_impl.h:85]: Oh no, I was slightly too fast!

Comment (matt-attack) [reply swri_roscpp/include/swri_roscpp/subscriber_impl.h:85]: You're just too responsive!

**Pull request merged**